src/pages/DefaultPage/DefaultPage.tsx

```diff
@@ -57,47 +53,24 @@ const useStyles = makeStyles(theme => ({
 export const DefaultPage = () => {
   const classes = useStyles();
   const navigate = useNavigate();
-  const web3Context = useWeb3React<Web3Provider>();
 
-  const authToken = useAuthToken();
   const myProfile = useRecoilValueLoadable(rMyProfile).valueMaybe();
   const selectedCircle = useRecoilValueLoadable(rSelectedCircle).valueMaybe();
 
-  const Wrapper = ({ children }: { children: ReactNode }) => (
-    <Box
-      css={{
-        maxWidth: '700px',
-        mx: 'auto',
-        pt: '$2xl',
-        px: '$lg',
-        textAlign: 'center',
-      }}
-    >
-      {children}
-      <Footer />
-    </Box>
-  );
-
-  // TODO: Split these off into separate components..
-  // But also Alex Ryan likes the idea of us making this more useful.
-  if (!authToken) {
-    return (
-      <Wrapper>
-        <p className={classes.title}>Reward Your Fellow Contributors</p>
-        <p className={classes.subTitle}>
-          {!web3Context.account
-            ? 'Connect your wallet to participate.'
-            : 'Login to Coordinape'}
-        </p>
-      </Wrapper>
-    );
-  }
   // still loading
   if (!myProfile) return null;
 
   if (!selectedCircle) {
     return (
-      <Wrapper>
```

Comment:
need to fix this so that if there is some other circle you could be in, just not selected, you get redirected to /circles